### PR TITLE
Added glyph and drop animation to group header

### PR DESCRIPTION
**Pull request by wdo3650**
Added glyph and animation to GroupedList headers for drag and drop.  I also changed the size of the chevron down per design guidance.

### CODE REVIEW

File: src/common/_tempMDL2Icons.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
...8AZwByAGEAcABoAHkALwBmAG8AbgB0AHMALwBZAG8AdQAgAG0AYQB5ACAAdQBzAGUAIAB0AGgAaQBzACAAZgBvAG4AdAAgAGEAcwAgAHAAZQByAG0AaQB0AHQAZQBkACAAYgB5ACAAdABoAGUAIABFAFUATABBACAAZgBvAHIAIAB0AGgAZQAgAHAAcgBvAGQAdQBjAHQAIABpAG4AIAB3AGgAaQBjAGgAIAB0AGgAaQBzACAAZgBvAG4AdAAgAGkAcwAgAGkAbgBjAGwAdQBkAGUAZAAgAHQAbwAgAGQAaQBzAHAAbABhAHkAIABhAG4AZAAgAHAAcgBpAG4AdAAgAGMAbwBuAHQAZQBuAHQALgAgAFkAbwB1ACAAbQBhAHkAIABvAG4AbAB5ACAAKABpACkAIABlAG0AYgBlAGQAIAB0AGgAaQBzACAAZgBvAG4AdAAgAGkAbgAgAGMAbwBuAHQAZQBuAHQAIABhAHMAIABwAGUAcgBtAGkAdAB0AGUAZAAgAGIAeQAgAHQAaABlACAAZQBtAGIAZQBkAGQAaQBuAGcAIAByAGUAcwB0AHIAaQBjAHQAaQBvAG4AcwAgAGkAbgBjAGwAdQBkAGUAZAAgAGkAbgAgAHQAaABpAHMAIABmAG8AbgB0ADsAIABhAG4AZAAgACgAaQBpACkAIAB0AGUAbQBwAG8AcgBhAHIAaQBsAHkAIABkAG8AdwBuAGwAbwBhAGQAIAB0AGgAaQBzACAAZgBvAG4AdAAgAHQAbwAgAGEAIABwAHIAaQBuAHQAZQByACAAbwByACAAbwB0AGgAZQByACAAbwB1AHQAcAB1AHQAIABkAGUAdgBpAGMAZQAgAHQAbwAgAGgAZQBsAHAAIABwAHIAaQBuAHQAIABjAG8AbgB0AGUAbgB0AC4ATgBvAHIAbQBhAGEAbABpAE4AbwByAG0A4QBsAE4AbwByAG0AYQBsAGUAUwB0AGEAbgBkAGEAYQByAGQATgBvAHIAbQBhAGwAbgB5BB4EMQRLBEcEPQRLBDkATgBvAHIAbQDhAGwAbgBlAE4AYQB2AGEAZABuAG8AdABoAbABoQMAAG4AZwBBAHIAcgB1AG4AdABhAAMAAAAAAAD/TgB3AAAAAAAAAAAAAAAAAAAAAAAAAAAAAQACAAgACv//AA8=') format('truetype');
```
Comment (dzearing) [inline src/common/_tempMDL2Icons.scss]: Ugh. This seems very bad. It will increase the download size significantly...

Comment (atneik) [inline src/common/_tempMDL2Icons.scss]: The other way to avoid this will be to have MDL2 beta merged in master, as it has MDL2 icons.

Comment (dzearing) [inline src/common/_tempMDL2Icons.scss]: I don't think we should take this. This will impact existing usage of details list.

Comment (dzearing) [inline src/common/_tempMDL2Icons.scss]: If you just needed 1 icon, couldn't you simply embed the font containing a single icon here?

It seems like you're embedding ALL of the icons here. This really isn't the right place to do this either, it should be done in products. Doing it here means everyone will merge this into their javascript, including products that have their own icon fonts.

Comment (atneik) [inline src/common/_tempMDL2Icons.scss]: @wdo3650 Wasn't this just a single Tag icon?

Comment (wdo3650) [inline src/common/_tempMDL2Icons.scss]: @atneik affirmative, this is just one icon.

File: src/components/GroupedList/GroupHeader.scss [hunk 2]
```diff
@@ -24,15 +28,17 @@
   cursor: default;
   padding: 8px;
   transform: translateY(50%);
-  margin-top: -10px;
+  margin-top: -12px;
   box-sizing: border-box;
   vertical-align: top;
   background: none;
   border: none;
+  font-size: 20px;
```
Comment (dzearing) [inline src/components/GroupedList/GroupHeader.scss]: Shouldn't the font-size use a sass variable?

File: src/components/GroupedList/GroupHeader.scss [hunk 3]
```diff
@@ -80,3 +86,36 @@
     opacity: 1;
   }
 }
+
+.ms-GroupHeader-dropIcon {
+  display: inline-block;
+  position: relative;
+  top: -16px;
+  left: -26px;
+  font-size: 20px;
```
Comment (dzearing) [inline src/components/GroupedList/GroupHeader.scss]: same here.

File: src/components/GroupedList/GroupHeader.scss [hunk 4]
```diff
@@ -80,3 +86,36 @@
...
+  transform-origin: 10px 10px;
+
+  .ms-Icon--tag {
+    position: absolute;
+  }
+}
+
+.ms-GroupedList-group.is-dropping {
+   > .ms-GroupHeader {
+      .ms-GroupHeader-dropIcon {
```
Comment (dzearing) [inline src/components/GroupedList/GroupHeader.scss]: any reason to not just collapse this into one statement?

Comment (wdo3650) [inline src/components/GroupedList/GroupHeader.scss]: do you mean collapsing the nested selector like this?

> .ms-GroupHeader .ms-GroupHeader-dropIcon { ...

Comment (dzearing) [inline src/components/GroupedList/GroupHeader.scss]: yup

**Comment by wdo3650** [comment]
Merge issues with other branches. Will reattempt.

**Pull request closed without merging**